src/hacbs/components/topology/factories/VisualizationFactory.tsx

```diff
@@ -80,18 +84,26 @@ const VisualizationFactory: React.FC<VisualizationFactoryProps> = ({
       controller.fromModel(model, model.graph.layout === layoutRef.current);
       layoutRef.current = model.graph.layout;
       controller.getGraph().layout();
+      if (fitToScreen) {
+        controller.getGraph().fit(90);
+      }
```

Comment:
I don't see `fitToScreen` being used by any caller of this function.

Furthermore if `fitToScreen` changes, this effect does a whole lot more than what's needed to fit to screen. 

If you still need this prop, move this to a separate effect.

Reply:
I removed it as it is not used in other graphs